ds_judgements_public_ui/templates/includes/header_nav.html

```diff
@@ -9,19 +9,34 @@
       <ol>
         <li>
           <span class="page-header__breadcrumb-you-are-in">You are in:</span>
-          {% if current == "home" %}
+          {% if title == "home" %}
             {% translate "common.findcaselaw" %}
           {% else %}
             <a href="{% url 'home' %}">{% translate "common.findcaselaw" %}</a>
           {% endif %}
         </li>
         {% if search_context %}
           <li>
-            <a href="{{ search_context.search_url }}">{% translate "breadcrumbs.search" %}
-              {% if search_context.query %}for "{{ search_context.query }}"{% endif %}</a>
+            <a href="{{ search_context.search_url }}">
+              {% translate "breadcrumbs.search" %}
+              {% if search_context.query %}for "{{ search_context.query }}"{% endif %}
+              </a>
             </li>
           {% endif %}
-          {% if title and link %}<li>{{ title }}</li>{% endif %}
+          {% if link %}
```

Comment:
title being truthy was redundant here - we're never going to have a valid document with empty string as the FRBRName - or would we!?

Can add back in if we think we need. Will think of a test for it..

Reply:
Manually set the FRBRName of a press summary on staging to "" (empty string) and it looks like this. <img width="856" alt="Screenshot 2023-07-05 at 17 59 54" src="https://github.com/nationalarchives/ds-caselaw-public-ui/assets/42998618/25a5f9f3-d989-438f-a97e-272985531226">

We should never get to this scenario though since we should not be publishing docs without names! And its better we see the PUI as obviously broken with these weird breadcrumb arrows with missing name, than trying to make it look nice in that case!